### DISCUSSION TITLE
Percentage bypass permission for drop mob spawners OR Allways drop mob spawners placed by players

### DISCUSSION THREAD

**Discussion by ghost** [Zrips/CMI#3728] [Feature Requests]
###- Detailed explanation of what it should do, when and in what situation: ###
You can have a custom percentage on if a spawner will drop or not when silk touched. I would like a permission that allows you to bypass that set percentage. Or a check that allways drops the mob spawner when silk touch is used if the spawner was placed by a player on the terrain. For example You place a spawner you just picked up from the world and you want to pick it up, but not have a chance of losing it.

### Basic explanation as to why this feature is needed and/or useful to have: ###
Would be great for donor ranks or so certain people can allways get a spawner they silk touch instead of the set custom percentage.

**Comment by CloudeLecaw** [comment]
You can define this allready in the config. Look for the config spawner break section
You just need to enable it in config and give your groups the right permission for it (look in config section for it)

**Comment by ghost** [comment]
> You can define this allready in the config. Look for the config spawner break section
> You just need to enable it in config and give your groups the right permission for it (look in config section for it)

There is no permission to bypass custom spawner drop percentages. And there is no permission to allways break player placed spawners. The permission for being allowed to break spawners handles all spawners in the world and is effected by your custom spawner drop percentage. Not just player placed ones.
